content/blog/2021-08-16-august-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,258 @@
+[Bring DevOps to Data Science with MLOps](https://www.analyticsvidhya.com/blog/2021/04/bring-devops-to-data-science-with-continuous-mlops/)
+Amit walks through a tutorial using CML to bring CI/CD functionality to your ML
+project and automate the process. All great posts to check out!👇🏼
+
+<external-link
+href="https://www.analyticsvidhya.com/blog/2021/06/mlops-tracking-ml-experiments-with-data-version-control/?utm_source=dlvr.it&utm_medium=twitter"
+title="Tracking ML Experiments With Data Version Control"
+description="Amit Kulkarni's tutorial on getting started with DVC and tracking eperiments"
+link="https://analyticsvidhya.com"
+image="/uploads/images/2021-08-16/a-v.png"/> <external-link
+href="https://www.analyticsvidhya.com/blog/2021/06/mlops-versioning-datasets-with-git-dvc/"
+title="MLOps | Versioning Datasets with Git & DVC"
+description="Amit Kulkarni's tutorial on how to DVC works with Git to version your datasets."
+link="https://analyticsvidhya.com"
+image="/uploads/images/2021-08-16/a-v.png"/> <external-link
+href="https://www.analyticsvidhya.com/blog/2021/04/bring-devops-to-data-science-with-continuous-mlops/"
+title="Bring DevOps To Data Science With MLOps"
+description="Amit Kulkarni's tutorial on how to use CML to bring the CI/CD functionality of DevOps to your data science projects."
+link="https://analyticsvidhya.com"
+image="/uploads/images/2021-08-16/a-v.png"/>
+
+## Andreas Malekos' MLOps Tool Stack at Continuum Industries
+
+Last but not least, we bring you a great article from
+[Andreas Malekos](https://www.linkedin.com/in/andreasmalekos/), Chief Scientist
+at [Continuum Industries](https://www.continuum.industries/). In
+[the post](https://neptune.ai/blog/mlops-tool-stack-continuum-industries) he
+outlines the tool stack and MLOPs platform they use to do their work automating
+and optimizing the design of linear infrastructure assets like water pipelines,
+overhead transmission lines, subsea power lines, or telecommunication cables.
+
+Amongst their tool stack are DVC and CML, and the article outlines what they
+like (!🙈Spoiler alert🙊! DVC making repeatability achievable) and the things
+that they don't like that still need to be improved.
+
+![Continuum Industries MLOps Tool Stack](/uploads/images/2021-08-16/continuum-tool-stack.png)
+_Continuum Industries MLOps Tool Stack
+[Source link](https://neptune.ai/wp-content/uploads/Continuum-Industries-tool-stack-final.png)_
+
+# DVC News
+
+Though the team has been taking some vacation time in the last month, there's
+still a lot going on!
+
+![Typing Cat](https://media.giphy.com/media/aNqEFrYVnsS52/giphy.gif)
+
+## Docs Updates
+
+This month we are introducing docs updates! We will be including this in each
+month's Heartbeat so that you will always be aware of the what has changed as
+our open source projects mature.
+
+Our docs team made up of
+[Jorge Orpinel](https://www.linkedin.com/in/jorgeorpinel/),
+[Emre Şahin](https://www.linkedin.com/in/iemresahin/),
```

Comment:
It's better to give my https://emresahin.net link here. I don't use that Linked.in page.